src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -2040,7 +2040,7 @@ internal override CharSet MarshallingCharSet
             }
         }
 
-        internal override bool IsSerializable
+        public override bool IsSerializable
```

Comment:
>public override bool IsSerializable [](start = 8, length = 35)

We could keep the original accessibility and add explicit implementation, but it is fine to keep this change.